src/main/java/net/openhft/chronicle/threads/VanillaEventLoop.java

```diff
@@ -94,7 +94,7 @@ public String toString() {
     public void addHandler(@NotNull final EventHandler handler) {
         throwExceptionIfClosed();
 
-        checkInterrupted();
+        checkInterruptedAddingNewHandler();
```

Comment:
Ditto here

Reply:
Yep - I agree. Keen to hear what you think @minborg. I suspect we can get rid of these checks altogether now that event loop/handler lifecycle is better controlled and tested for.

If we can't reach consensus I would be keen to merge this as-is so we have a more explanatory warning message, and can use this to de-flake some other tests

Reply:
I think there is still room for improvement in how handlers are added, lifecycle etc. 

In the code above, `checkInterrupted()` is testing if *the current thread* (and thus possibly not the event thread) is interrupted. So, if another thread is adding a handler to the event thread, then we check if the calling thread is interrupted and not the event thread. The do-while loop is always testing a thread that is not the event loop thread. 

So, the first `checkInterrupt()` is redundant because if it is the event thread that adds a handler, the likelihood of interruption is very low and if it is an external thread, why should we check the interrupt status here?

Reply:
So, remove